<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchEventDialog</class>
 <widget class="QWidget" name="SearchEventDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Event</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_text"/>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="combo_command">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="SwitchVariableWidget" name="switch_widget" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radio_text">
       <property name="text">
        <string>Text</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radio_switch">
       <property name="text">
        <string>Switch</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item row="3" column="1">
      <widget class="SwitchVariableWidget" name="variable_widget" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="radio_variable">
       <property name="text">
        <string>Variable</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="radio_command">
       <property name="text">
        <string>Command</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>Texts in Non-Text Commands</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string>Ignore Case</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchVariableWidget</class>
   <extends>QWidget</extends>
   <header>dialogs/switchvariablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radio_switch</sender>
   <signal>toggled(bool)</signal>
   <receiver>switch_widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_variable</sender>
   <signal>toggled(bool)</signal>
   <receiver>variable_widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_command</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_command</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_text</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_text</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_text</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_text</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
